src/base/rss/rss_feed.cpp

```diff
@@ -212,7 +212,13 @@ void Feed::handleParsingFinished(const RSS::Private::ParsingResult &result)
         m_lastBuildDate = result.lastBuildDate;
 
         int newArticlesCount = 0;
-        for (const QVariantHash &varHash : result.articles) {
+        const QDateTime now {QDateTime::currentDateTime()};
+        for (QVariantHash varHash : result.articles) {
+            // if article has no publication date we use feed update time as a fallback
```

Comment:
Will this solve the bug? Won't this re-update the pud date on the next refresh? Shouldn't it be static after the 1st parse?

Reply:
>Shouldn't it be static after the 1st parse?

Yes. It's static. We never update existing articles from the source.